### ISSUE TITLE
Using 3.12.0, software simply crashes

### ISSUE THREAD

**Issue by SirCameron** [prisma/prisma#12684]
### Bug description

After installing @prisma/client and prisma CLI at version 3.12.0, node simply crashes without any output.
the code works when removing: 
```
const { PrismaClient } = require("@prisma/client");
const prisma = new PrismaClient();
```
So it's clearly a Prisma issue, but there is not output to trace it.

`prisma generate` works fine without errors.

### How to reproduce

1. install version 3.12.0
2. add prisma client after generating 
3. crash.

### Expected behavior

no crash. code should run.

### Prisma information
```prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}

model Account {
  id             Int       @id @default(autoincrement()) @db.UnsignedInt
  profile_id     Int       @db.UnsignedInt
  name           String    @default("") @db.VarChar(255)
  color          String?   @default("") @db.VarChar(100)
  initialBalance Decimal   @default(0.00) @db.Decimal(10, 10)
  currency       String    @default("USD") @db.Char(3)
  created_at     DateTime? @db.DateTime(0)
  updated_at     DateTime? @db.DateTime(0)

  Transaction Transaction[] @relation("account")
  @@map("accounts")
}

model Balance {
  id         Int       @id @default(autoincrement()) @db.UnsignedInt
  account_id BigInt    @db.UnsignedBigInt
  year       Int       @db.UnsignedInt
  balance    Decimal   @db.Decimal(10, 2)
  updated_at DateTime? @db.DateTime(0)

  @@map("balances")
}

model Category {
  id                   Int           @id @default(autoincrement()) @db.UnsignedInt
  profile_id           BigInt        @db.UnsignedBigInt
  name                 String        @default("") @db.VarChar(255)
  isBusinessExpense    Int           @default(0) @db.UnsignedTinyInt
  percentageDeductable Decimal       @default(0.00) @db.Decimal(10, 2)
  created_at           DateTime?     @db.DateTime(0)
  updated_at           DateTime?     @db.DateTime(0)
  Transaction          Transaction[] @relation("category")
}

model Project {
  id         Int       @id @default(autoincrement()) @db.UnsignedInt
  profile_id Int       @db.UnsignedInt
  name       String    @default("") @db.VarChar(255)
  created_at DateTime? @db.DateTime(0)
  updated_at DateTime? @db.DateTime(0)

  @@map("projects")
}

model Subject {
  id         Int      @id @default(autoincrement()) @db.UnsignedInt
  profile_id Int      @db.UnsignedInt
  name       String   @default("") @db.VarChar(255)
  created_at DateTime @db.DateTime(0)
  updated_at DateTime @db.DateTime(0)

  Transaction Transaction[] @relation("subject")
  @@map("subjects")
}

model Transaction {
  id             Int          @id @default(autoincrement()) @db.UnsignedInt
  profile_id     Int          @db.UnsignedInt
  account_id     Int          @db.UnsignedInt
  account        Account      @relation("account", fields: [account_id], references: [id])
  category_id    Int?         @db.UnsignedInt
  category       Category?    @relation("category", fields: [category_id], references: [id])
  subject_id     Int?         @db.UnsignedInt
  subject        Subject?     @relation("subject", fields: [subject_id], references: [id])
  date           DateTime?    @db.Date
  amount         Int          @default(0)
  vat            Int          @default(0)
  note           String?      @db.VarChar(255)
  transaction_id Int?         @unique @db.UnsignedInt
  transfer       Transaction? @relation("transfer", fields: [transaction_id], references: [id])
  transaction    Transaction? @relation("transfer")
  is_transfer    Int          @default(0)

  created_at DateTime? @db.DateTime(0)
  updated_at DateTime? @db.DateTime(0)

  @@map("transactions")
}

model UserProfile {
  id         Int     @id @default(autoincrement()) @db.UnsignedInt
  user_id    Int     @db.UnsignedInt
  profile_id Int     @db.UnsignedInt
  profiles   Profile @relation("profilesTouser_profiles", fields: [profile_id], references: [id])
  users      User    @relation("user_profilesTousers", fields: [user_id], references: [id])

  @@index([profile_id], name: "fk_profiles")
  @@index([user_id], name: "fk_users")
  @@map("user_profiles")
}

model User {
  id            Int           @id @default(autoincrement()) @db.UnsignedInt
  email         String        @unique @default("") @db.VarChar(255)
  name          String?       @db.VarChar(255)
  password      String?       @db.VarChar(255)
  created_at    DateTime      @db.DateTime(0)
  updated_at    DateTime      @db.DateTime(0)
  user_profiles UserProfile[] @relation("user_profilesTousers")

  @@map("users")
}

model Profile {
  id            Int           @id @default(autoincrement()) @db.UnsignedInt
  name          String        @default("") @db.VarChar(255)
  created_at    DateTime?     @db.DateTime(0)
  updated_at    DateTime?     @db.DateTime(0)
  user_profiles UserProfile[] @relation("profilesTouser_profiles")

  @@map("profiles")
}
```

### Environment & setup

- OS: mac os 12.3
- Database: mysql 8.0.24
- Node.js version: v16.13.2

### Prisma Version

as mentioned, 3.12.0

**Comment by garrensmith** [comment]
@SirCameron thanks for the error report. But I cannot reproduce the issue. Just confirming you running `npx prisma generate` ?
Could you try that schema on a new project and let me know the exact steps you are doing to set it up.

**Comment by Simskii** [comment]
Same here, using node v16.13.1.
App crashes at startup.

**Comment by garrensmith** [comment]
@Simskii could you provide a simple reproduction for us to take a look at?

**Comment by PatricioNG** [comment]
I believe I just encountered the same issue @garrensmith, after starting my backend dev server on 3.12.0 I noticed after around 7-10 minutes, I received this error:
`
<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 00007FF6C6A74C6F napi_wrap+111007
 2: 00007FF6C6A181F6 v8::base::CPU::has_sse+59910
 3: 00007FF6C6A190F6 node::OnFatalError+294
 4: 00007FF6C72F220E v8::Isolate::ReportExternalAllocationLimitReached+94
 5: 00007FF6C72D6FDD v8::SharedArrayBuffer::Externalize+781
 6: 00007FF6C718011C v8::internal::Heap::EphemeronKeyWriteBarrierFromCode+1516
 7: 00007FF6C718B53A v8::internal::Heap::ProtectUnprotectedMemoryChunks+1258
 8: 00007FF6C7188679 v8::internal::Heap::PageFlagsAreConsistent+2457
 9: 00007FF6C717D211 v8::internal::Heap::CollectGarbage+2049
10: 00007FF6C717B415 v8::internal::Heap::AllocateExternalBackingStore+1349
11: 00007FF6C719B88B v8::internal::Factory::NewFillerObject+203
12: 00007FF6C6ECA361 v8::internal::interpreter::JumpTableTargetOffsets::iterator::operator=+1409
13: 00007FF6C737B0CD v8::internal::SetupIsolateDelegate::SetupHeap+465325
14: 00007FF6C73CC34F v8::internal::SetupIsolateDelegate::SetupHeap+797743
15: 00007FF6C736BEC2 v8::internal::SetupIsolateDelegate::SetupHeap+403362
16: 0000007B80E95571`

So I then started it again and this time had no jobs/tasks/processes running, just the api server on it's own and I watched node's memory usage slowly crawl to full usage until there was no memory left and then received the same error again:

`<--- Last few GCs --->

[5992:00000262EF8627F0]   336261 ms: Mark-sweep (reduce) 2043.8 (2050.5) -> 2042.9 (2051.5) MB, 1356.1 / 0.0 ms  (average mu = 0.167, current mu = 0.031) allocation failure scavenge might not succeed
[5992:00000262EF8627F0]   337956 ms: Mark-sweep (reduce) 2043.9 (2053.5) -> 2043.0 (2053.7) MB, 1674.5 / 0.0 ms  (average mu = 0.086, current mu = 0.013) allocation failure scavenge might not succeed


<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 00007FF6C6A74C6F napi_wrap+111007
 2: 00007FF6C6A181F6 v8::base::CPU::has_sse+59910
 3: 00007FF6C6A190F6 node::OnFatalError+294
 4: 00007FF6C72F220E v8::Isolate::ReportExternalAllocationLimitReached+94
 5: 00007FF6C72D6FDD v8::SharedArrayBuffer::Externalize+781
 6: 00007FF6C718011C v8::internal::Heap::EphemeronKeyWriteBarrierFromCode+1516
 7: 00007FF6C718B53A v8::internal::Heap::ProtectUnprotectedMemoryChunks+1258
 8: 00007FF6C7188679 v8::internal::Heap::PageFlagsAreConsistent+2457
 9: 00007FF6C717D211 v8::internal::Heap::CollectGarbage+2049
10: 00007FF6C717B415 v8::internal::Heap::AllocateExternalBackingStore+1349
11: 00007FF6C719B88B v8::internal::Factory::NewFillerObject+203
12: 00007FF6C6ECA361 v8::internal::interpreter::JumpTableTargetOffsets::iterator::operator=+1409
13: 00007FF6C737B0CD v8::internal::SetupIsolateDelegate::SetupHeap+465325
14: 000003B0222CE4FA`

Hope this is enough to go off of - Seems to be repeatable for me - start Node, wait 7-10 minutes, crash 🙏

**Comment by garrensmith** [comment]
Thanks for the repo but unfortunately that still doesn’t give enough information. Are you able to create a simplified reproduction that I could run locally?
________________________________

**Comment by janpio** [comment]
@SirCameron How are you running your application? Does it crash instantly or over time?
@Simskii How are you running your application?
@PatricioNG How do you start your application? Does it do anything in idle if no requests come in?

And for everyone: 
Are these all applications that were upgraded from a previous Prisma version? Which one? I assume you did generate a new Prisma Client after the upgrade - manually or behind the scenes? Are you generating your Client to a custom location with `output`? Is this also reproducible if you create a new project with `npx prisma init` in an empty folder?

This sounds bad, but right now we are indeed struggling to reproduce. For us everything works as it should, but we totally believe you that this is not true for you (at no fault of yours) and want to understand and fix.

**Comment by SirCameron** [comment]
@janpio thanks. I'm just running `node index.js`..
I got this working by deleting node_modules and reinstalling... obviously some old trash was still in there causing issues.
all working now.
Hopefully this can help the others having the same issue.

Thanks @janpio @garrensmith for your help!

**Comment by janpio** [comment]
Ok, that would have been my next suggestion as a potential workaround or solution.

For anyone else who has the same problem: **Please** zip your `node_modules` folder into an archive before trying this! We would _really_ love to be able to compare a before and after of a `node_modules` folder where this happens, and deleting `node_modules` turns out to be the solution. This is what could help us identify the problem and hopefully fix it. Thanks.

@PatricioNG Based on this information, I think your problem is different from what the other 2 reported and you could optimally create a new bug report with all the information about this that could be useful. Thanks.

**Comment by PatricioNG** [comment]
Hi @janpio - Thanks for responding, here's the answers to your question off the side of my desk, I'll try and see if I can get a small repro up and running later this afternoon 

- How do you start your application?
For me, this is just in my dev environment so I have a start script that runs `npx tsc` then cd's into the build folder and runs `node server.js`

- Does it do anything in idel if no requests come in?
No, in my dev environment all batch jobs are turned off, so if the server is running it's simply waiting for requests

- Was it upgraded from a previous Prisma version?
Yes `"@prisma/client": "^3.8.1"`

- I assume you did generate a new prisma client after the upgrade
Yes, correct I have a script that runs post install things for me that includes `npx prisma generate --schema ./prisma/EcommSchema.prisma`

- Am I generating my client to a custom location with `output`
Yes, looks like: ` output   = "./generated/EcommSchema"`

If I get time will create a new project this evening and see if I can reproduce

**Comment by Simskii** [comment]
Could not reproduce in a new project.
After removing node_modules & yarn.lock all issues were solved.
Tried downgrading to 3.10 and upgrading to 3.12 again but the error does not occur anymore.

**Comment by janpio** [comment]
Did you by chance still have a backup of the broken `node_modules` folder @Simskii?

**Comment by Simskii** [comment]
Unfortunately not... @janpio

**Comment by janpio** [comment]
@Simskii did find another case and send me both `node_modules` folder. They are mostly identical, besides tiny, irrelvant differences in the newer versions of some random apcakges of his project, and a difference in the `.yarn-integrity` file. Here is a unified diff:

```diff
--- C:/Users/Jan/Downloads/broken_node_modules/node_modules/.yarn-integrity	Fri Apr  8 15:17:06 2022
+++ C:/Users/Jan/Downloads/working_node_modules/node_modules/.yarn-integrity	Fri Apr  8 17:34:51 2022
@@ -428,7 +428,7 @@
     "autolinker@~0.28.0": "https://registry.yarnpkg.com/autolinker/-/autolinker-0.28.1.tgz#0652b491881879f0775dace0cdca3233942a4e47",
     "autoprefixer@^9.4.5": "https://registry.yarnpkg.com/autoprefixer/-/autoprefixer-9.8.8.tgz#fd4bd4595385fa6f06599de749a4d5f7a474957a",
     "awesome-phonenumber@^2.2.6": "https://registry.yarnpkg.com/awesome-phonenumber/-/awesome-phonenumber-2.70.0.tgz#81ed3abd1cecd08f974563b0973275f4c0f51efc",
-    "aws-sdk@^2.384.0": "https://registry.yarnpkg.com/aws-sdk/-/aws-sdk-2.1109.0.tgz#1be11188303f6545891edb0f7facf295f3ff9a40",
+    "aws-sdk@^2.384.0": "https://registry.yarnpkg.com/aws-sdk/-/aws-sdk-2.1110.0.tgz#bb4fe12a7768957b8df275bcd5e4188f5b5a9393",
     "aws-sign2@~0.7.0": "https://registry.yarnpkg.com/aws-sign2/-/aws-sign2-0.7.0.tgz#b46e890934a9591f2d2f6f86d7e6a9f1b3fe76a8",
     "aws4@^1.8.0": "https://registry.yarnpkg.com/aws4/-/aws4-1.11.0.tgz#d61f46d83b2519250e2784daf5b09479a8b41c59",
     "axios-logger@^2.5.0": "https://registry.yarnpkg.com/axios-logger/-/axios-logger-2.6.0.tgz#8baa8164846481530b1f822bf760eaf004d33d98",
@@ -520,8 +520,8 @@
     "camelcase@^5.0.0": "https://registry.yarnpkg.com/camelcase/-/camelcase-5.3.1.tgz#e3c9b31569e106811df242f715725a1f4c494320",
     "camelcase@^5.3.1": "https://registry.yarnpkg.com/camelcase/-/camelcase-5.3.1.tgz#e3c9b31569e106811df242f715725a1f4c494320",
     "camelcase@^6.0.0": "https://registry.yarnpkg.com/camelcase/-/camelcase-6.3.0.tgz#5685b95eb209ac9c0c177467778c9c84df58ba9a",
-    "caniuse-lite@^1.0.30001109": "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001325.tgz#2b4ad19b77aa36f61f2eaf72e636d7481d55e606",
-    "caniuse-lite@^1.0.30001317": "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001325.tgz#2b4ad19b77aa36f61f2eaf72e636d7481d55e606",
+    "caniuse-lite@^1.0.30001109": "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001327.tgz#c1546d7d7bb66506f0ccdad6a7d07fc6d668c858",
+    "caniuse-lite@^1.0.30001317": "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001327.tgz#c1546d7d7bb66506f0ccdad6a7d07fc6d668c858",
     "capture-exit@^2.0.0": "https://registry.yarnpkg.com/capture-exit/-/capture-exit-2.0.0.tgz#fb953bfaebeb781f62898239dabb426d08a509a4",
     "capture-stack-trace@^1.0.0": "https://registry.yarnpkg.com/capture-stack-trace/-/capture-stack-trace-1.0.1.tgz#a6c0bbe1f38f3aa0b92238ecb6ff42c344d4135d",
     "caseless@~0.12.0": "https://registry.yarnpkg.com/caseless/-/caseless-0.12.0.tgz#1b681c21ff84033c826543090689420d187151dc",
@@ -730,7 +730,7 @@
     "ecc-jsbn@~0.1.1": "https://registry.yarnpkg.com/ecc-jsbn/-/ecc-jsbn-0.1.2.tgz#3a83a904e54353287874c564b7549386849a98c9",
     "ecdsa-sig-formatter@1.0.11": "https://registry.yarnpkg.com/ecdsa-sig-formatter/-/ecdsa-sig-formatter-1.0.11.tgz#ae0f0fa2d85045ef14a817daa3ce9acd0489e5bf",
     "ee-first@1.1.1": "https://registry.yarnpkg.com/ee-first/-/ee-first-1.1.1.tgz#590c61156b0ae2f4f0255732a158b266bc56b21d",
-    "electron-to-chromium@^1.4.84": "https://registry.yarnpkg.com/electron-to-chromium/-/electron-to-chromium-1.4.105.tgz#30de5e4ba020140b698539b2d366cd9c3a337ec7",
+    "electron-to-chromium@^1.4.84": "https://registry.yarnpkg.com/electron-to-chromium/-/electron-to-chromium-1.4.106.tgz#e7a3bfa9d745dd9b9e597616cb17283cc349781a",
     "email-validator@^2.0.4": "https://registry.yarnpkg.com/email-validator/-/email-validator-2.0.4.tgz#b8dfaa5d0dae28f1b03c95881d904d4e40bfe7ed",
     "emittery@^0.7.1": "https://registry.yarnpkg.com/emittery/-/emittery-0.7.2.tgz#25595908e13af0f5674ab419396e2fb394cdfa82",
     "emoji-regex@^8.0.0": "https://registry.yarnpkg.com/emoji-regex/-/emoji-regex-8.0.0.tgz#e818fd69ce5ccfcb404594f842963bf53164cc37",
@@ -745,8 +745,8 @@
     "error-symbol@^0.1.0": "https://registry.yarnpkg.com/error-symbol/-/error-symbol-0.1.0.tgz#0a4dae37d600d15a29ba453d8ef920f1844333f6",
     "es-abstract@^1.19.1": "https://registry.yarnpkg.com/es-abstract/-/es-abstract-1.19.2.tgz#8f7b696d8f15b167ae3640b4060670f3d054143f",
     "es-to-primitive@^1.2.1": "https://registry.yarnpkg.com/es-to-primitive/-/es-to-primitive-1.2.1.tgz#e55cd4c9cdc188bcefb03b366c736323fc5c898a",
-    "es5-ext@^0.10.35": "https://registry.yarnpkg.com/es5-ext/-/es5-ext-0.10.59.tgz#71038939730eb6f4f165f1421308fb60be363bc6",
-    "es5-ext@^0.10.50": "https://registry.yarnpkg.com/es5-ext/-/es5-ext-0.10.59.tgz#71038939730eb6f4f165f1421308fb60be363bc6",
+    "es5-ext@^0.10.35": "https://registry.yarnpkg.com/es5-ext/-/es5-ext-0.10.60.tgz#e8060a86472842b93019c31c34865012449883f4",
+    "es5-ext@^0.10.50": "https://registry.yarnpkg.com/es5-ext/-/es5-ext-0.10.60.tgz#e8060a86472842b93019c31c34865012449883f4",
     "es6-denodeify@^0.1.5": "https://registry.yarnpkg.com/es6-denodeify/-/es6-denodeify-0.1.5.tgz#31d4d5fe9c5503e125460439310e16a2a3f39c1f",
     "es6-iterator@^2.0.3": "https://registry.yarnpkg.com/es6-iterator/-/es6-iterator-2.0.3.tgz#a7de889141a05a94b0854403b2d0a0fbfa98f3b7",
     "es6-promise@^4.0.3": "https://registry.yarnpkg.com/es6-promise/-/es6-promise-4.2.8.tgz#4eb21594c972bc40553d276e510539143db53e0a",
@@ -1340,7 +1340,7 @@
     "lru-cache@^4.0.1": "https://registry.yarnpkg.com/lru-cache/-/lru-cache-4.1.5.tgz#8bbe50ea85bed59bc9e33dcab8235ee9bcf443cd",
     "lru-cache@^4.1.3": "https://registry.yarnpkg.com/lru-cache/-/lru-cache-4.1.5.tgz#8bbe50ea85bed59bc9e33dcab8235ee9bcf443cd",
     "lru-cache@^6.0.0": "https://registry.yarnpkg.com/lru-cache/-/lru-cache-6.0.0.tgz#6d6fe6570ebd96aaf90fcad1dafa3b2566db3a94",
-    "lru-cache@^7.4.0": "https://registry.yarnpkg.com/lru-cache/-/lru-cache-7.7.3.tgz#98cd19eef89ce6a4a3c4502c17c833888677c252",
+    "lru-cache@^7.4.0": "https://registry.yarnpkg.com/lru-cache/-/lru-cache-7.8.0.tgz#649aaeb294a56297b5cbc5d70f198dcc5ebe5747",
     "make-dir@^1.0.0": "https://registry.yarnpkg.com/make-dir/-/make-dir-1.3.0.tgz#79c1033b80515bd6d24ec9933e860ca75ee27f0c",
     "make-dir@^3.0.0": "https://registry.yarnpkg.com/make-dir/-/make-dir-3.1.0.tgz#415e967046b3a7f1d185277d84aa58203726a13f",
     "make-error@1.x": "https://registry.yarnpkg.com/make-error/-/make-error-1.3.6.tgz#2eb2e37ea9b67c4891f684a1394799af484cf7a2",
@@ -2095,13 +2095,13 @@
   },
   "files": [],
   "artifacts": {
-    "@prisma/engines@3.10.0-50.73e60b76d394f8d37d8ebd1f8918c79029f0db86": [
+    "@prisma/engines@3.12.0-37.22b822189f46ef0dc5c5b503368d1bee01213980": [
       "introspection-engine-darwin-arm64",
       "libquery_engine-darwin-arm64.dylib.node",
       "migration-engine-darwin-arm64",
       "prisma-fmt-darwin-arm64"
     ],
-    "prisma@3.10.0": [
+    "prisma@3.12.0": [
       "libquery_engine-darwin-arm64.dylib.node"
     ],
     "@apollo/protobufjs@1.2.2": [
@@ -2443,15 +2443,6 @@
       "lib/protocol/crypto/build/config.gypi",
       "lib/protocol/crypto/build/gyp-mac-tool",
       "lib/protocol/crypto/build/sshcrypto.target.mk"
-    ],
-    "@prisma/engines@3.12.0-37.22b822189f46ef0dc5c5b503368d1bee01213980": [
-      "introspection-engine-darwin-arm64",
-      "libquery_engine-darwin-arm64.dylib.node",
-      "migration-engine-darwin-arm64",
-      "prisma-fmt-darwin-arm64"
-    ],
-    "prisma@3.12.0": [
-      "libquery_engine-darwin-arm64.dylib.node"
     ]
   }
 }
\ No newline at end of file
```

Here only the interesting part:
```diff
--- C:/Users/Jan/Downloads/broken_node_modules/node_modules/.yarn-integrity	Fri Apr  8 15:17:06 2022
+++ C:/Users/Jan/Downloads/working_node_modules/node_modules/.yarn-integrity	Fri Apr  8 17:34:51 2022
@@ -2095,13 +2095,13 @@
   },
   "files": [],
   "artifacts": {
-    "@prisma/engines@3.10.0-50.73e60b76d394f8d37d8ebd1f8918c79029f0db86": [
+    "@prisma/engines@3.12.0-37.22b822189f46ef0dc5c5b503368d1bee01213980": [
       "introspection-engine-darwin-arm64",
       "libquery_engine-darwin-arm64.dylib.node",
       "migration-engine-darwin-arm64",
       "prisma-fmt-darwin-arm64"
     ],
-    "prisma@3.10.0": [
+    "prisma@3.12.0": [
       "libquery_engine-darwin-arm64.dylib.node"
     ],
     "@apollo/protobufjs@1.2.2": [
@@ -2443,15 +2443,6 @@
       "lib/protocol/crypto/build/config.gypi",
       "lib/protocol/crypto/build/gyp-mac-tool",
       "lib/protocol/crypto/build/sshcrypto.target.mk"
-    ],
-    "@prisma/engines@3.12.0-37.22b822189f46ef0dc5c5b503368d1bee01213980": [
-      "introspection-engine-darwin-arm64",
-      "libquery_engine-darwin-arm64.dylib.node",
-      "migration-engine-darwin-arm64",
-      "prisma-fmt-darwin-arm64"
-    ],
-    "prisma@3.12.0": [
-      "libquery_engine-darwin-arm64.dylib.node"
     ]
   }
 }
\ No newline at end of file
```

Anyone an idea if this could be connected? What does this mean exactly? An artifact of `3.10.0` being present although they  updated Prisma to 3.12.0 already seems weird. But is it?

**Comment by SevInf** [comment]
@pChausseC @SirCameron @Simskii are you on M1 macs by any chance?
If so, this could be #14058.
We implemented a workaround in 4.1.0. Could you check most recent Prisma version and see if it is still happens?

@PatricioNG your issue looks like something different than reported originally. Could you open a separate one and include a reproduction?

**Comment by pChausseC** [comment]
@SevInf yes I'm on M1 mac. Updated to @latest (4.3.1) without running into any issues. (haven't dealt with this issue since 3.12)

**Comment by SevInf** [comment]
I am going to close this since it likely fixed in 4.1.0. Feel free to reopen if you manage to reproduce it on newer Prisma version
